packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -225,6 +241,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         this.attachmentPOSTUrl = this.odspResolvedUrl.endpoints.attachmentPOSTStorageUrl;
         this.attachmentGETUrl = this.odspResolvedUrl.endpoints.attachmentGETStorageUrl;
         this.odspSummaryUploadManager = new OdspSummaryUploadManager(this.snapshotUrl, getStorageToken, logger, epochTracker);
+        this.hostPolicy.fetchBinarySnapshotFormat ??= gatesBinaryFormatSnapshot();
```

Comment:
I think we do modify policy somewhere else, but we do make a copy first. Should this code do the same (and be where we do such modification)?